copulas/multivariate/tree.py

```diff
@@ -299,13 +276,12 @@ def _build_first_tree(self):
                         adj_set.add((x, k))
             # find edge with maximum
             edge = sorted(adj_set, key=lambda e: neg_tau[e[0]][e[1]])[0]
-            name, theta = Bivariate.select_copula(
-                self.u_matrix[:, edge[0]], self.u_matrix[:, edge[1]])
-
-            tau_matrix = self.tau_matrix[edge[0], edge[1]]
-            new_edge = Edge(itr, edge[0], edge[1], tau_matrix, name, theta)
-            new_edge.parent.append(edge[0])
-            new_edge.parent.append(edge[1])
+            name, theta = Bivariate.select_copula(self.u_matrix[:, edge[0]],
+                                                  self.u_matrix[:, edge[1]])
+            left = min(edge[0], edge[1])
```

Comment:
If you intend to set the min value in 'edge` as left, and the max as right, you can simply do.

```python
left, right = sorted(edge)
```
You can even do it on Edge.__init__, accepting just two nodes in an iterable, and do the assignment on the class instantiation, and avoid the sorting before creating the Edge instance. Something like:

```python
class Edge(object):
    def __init__(self, nodes, *args, **kwargs):

        self.left, self.right = sorted(nodes)

edge = Edge([7,4])
assert edge.L == 4
assert edge.R ==7 
```